courses/free-courses-en.md

```diff
@@ -441,6 +441,7 @@
 * [Golang in under an hour](https://www.youtube.com/watch?v=N0fIANJkwic) - Eli Goldberg
 * [Golang Tutorial - Learn Go Programming Language](https://www.geeksforgeeks.org/golang-tutorial-learn-go-programming-language/) - GeeksforGeeks
 * [Gophercises: Free Coding Exercises for Budding Gophers](https://gophercises.com) - Jon Calhoun (email address *required*)
+* [Learn Go](https://golangr.com)
```

Comment:
Better use full title resource and make reference to the website as author. I can't find it real name.

```suggestion
* [Learn Go \| Learn Go Programming](https://golangr.com) - golangr.com
```

Reply:
Is there a mistype with the `\`?

Reply:
> Is there a mistype with the `\`?

Is the markdown escape character :wink:

Reply:
Makes me curious who is behind this; probably related to the book it affiliate-links to. It's not copied from anywhere else. but yes, for this "golangr.com" is a useful attribution - that's who the copyright is asserted for.

Reply:
> Makes me curious who is behind this; probably related to the book it affiliate-links to. It's not copied from anywhere else. but yes, for this "golangr.com" is a useful attribution - that's who the copyright is asserted for.

Maybe @Callebdoxley ?